enterprise_access/settings/base.py

```diff
@@ -369,3 +369,8 @@ def root(*path_fragments):
 BRAZE_API_URL = ''
 BRAZE_API_KEY = os.environ.get('BRAZE_API_KEY', '')
 BRAZE_APP_ID = os.environ.get('BRAZE_APP_ID', '')
+
+# shell_plus
+SHELL_PLUS_IMPORTS = [
```

Comment:
what's this?

Reply:
This automatically loads additional imports when using shell_plus. I found myself needing to import localized_utcnow in the shell so this makes life easier.